ethcore/src/client/client.rs

```diff
@@ -888,7 +887,7 @@ impl Client {
 	/// Import transactions from the IO queue
 	pub fn import_queued_transactions(&self, transactions: &[Bytes], peer_id: usize) -> usize {
 		trace!(target: "external_tx", "Importing queued");
-		let _timer = PerfTimer::new("import_queued_transactions");
+		trace_time!("import_queued_transactions");
```

Comment:
are we 100% sure that this actually creates a binding at this scope and it isn't dropped immediately?

Reply:
https://play.rust-lang.org/?gist=050a982e9374596fbf6b45375b8a1ada&version=stable

seems to indicate that it does, and it is hygienic.